### DISCUSSION TITLE
Converting of Lora Adapter failing: Embeddings is present in the adapter. This can be due to new tokens added during fine tuning

### DISCUSSION THREAD

**Discussion by eugeis** [ggml-org/llama.cpp#10014] [Q&A]
Hello,

If am I trying to convert a Lora Adapter based on LLama-3.1-8b to gguf, I am getting the error:

ERROR:lora-to-gguf:Unexpected name 'base_model.model.lm_head.weight': Not a lora_A or lora_B tensor
ERROR:lora-to-gguf:Embeddings is present in the adapter. This can be due to new tokens added during fine tuning
ERROR:lora-to-gguf:Hint: if you are using TRL, make sure not to call setup_chat_format()

We are using indeed in pre-training phase new tokens, lm_head.

Any ideas how to solve the problem?

Thank you

https://github.com/ggerganov/llama.cpp/blob/c421ac072d46172ab18924e1e8be53680b54ed3b/convert_lora_to_gguf.py#L350

**Comment by eugeis** [comment]
Any ideas?

**Comment by AbnetS** [comment]
I am having the same problem.
